roles/kubernetes/preinstall/templates/ntp.conf.j2

```diff
@@ -35,6 +35,11 @@ restrict -6 default kod notrap nomodify nopeer noquery limited
 restrict {{ item }}
 {% endfor %}
 
+interface ignore wildcard
+{% for item in ntp_listen %}
+interface listen {{ item }}
+{% endfor %}
```

Comment:
To make it backwards compatible add a variable like `ntp_filter_interface` and set it to false and then:

```suggestion
{% if ntp_filter_interface %}
interface ignore wildcard
{% for item in ntp_listen %}
interface listen {{ item }}
{% endfor %}
{% endif %}
```